GRAPHICS/EnableAlienBloodVfx.md

```diff
@@ -9,10 +9,10 @@ aliases: ["0x9DCE1F0F78260875"]
 void ENABLE_ALIEN_BLOOD_VFX(BOOL toggle);
 ```
 
-Creates a motion-blur sort of effect, this native does not seem to work, however by using the [`START_SCREEN_EFFECT`](#_0x2206BF9A37B7F724) native with `"DrugsMichaelAliensFight"` as the effect parameter, you should be able to get the effect.
+Creates a motion-blur sort of effect, this native does not seem to work, however by using the [ANIMPOSTFX_PLAY](https://docs.fivem.net/natives/?_0x2206BF9A37B7F724) native with `"DrugsMichaelAliensFight"` as the effect parameter, you should be able to get the effect.
```

Comment:
Hey there, take a look at the following [line](https://github.com/citizenfx/natives/blob/b83b904e333fcd4ae80b5cf872899d8775d0be84/CAM/CreateCamWithParams.md?plain=1#L13) to see how this is done. It's better to not use domains and keep things relative, thank you for your contribution though!

Reply:
Will do thanks ! I did look in the natives website and many other natives were referenced using the full domain so that's why I did this